packages/subgraph/src/helpers/pair.ts

```diff
@@ -0,0 +1,48 @@
+import { Address } from '@graphprotocol/graph-ts'
+import { GardensTemplate as GardensTemplateContract } from '../../generated/GardensTemplate/GardensTemplate'
+import { HoneyswapFactory as HoneyswapFactoryContract } from '../../generated/GardensTemplate/HoneyswapFactory'
+import { HoneyswapRouter as HoneyswapRouterContract } from '../../generated/GardensTemplate/HoneyswapRouter'
+import { Pair as PairContract } from '../../generated/GardensTemplate/Pair'
+import { Pair as PairTemplate } from '../../generated/templates'
+import { loadOrCreateOrg, loadOrCreatePair, loadTokenData, ZERO_ADDRESS } from '.'
+import { Organization as OrganizationEntity } from '../../generated/schema'
+
+function populatePairDataFromContract(pairAddress: Address, org: OrganizationEntity | null): void {
+  const pairEntity = loadOrCreatePair(pairAddress)
+  const pair = PairContract.bind(pairAddress)
+
+  const token0Id = loadTokenData(pair.token0())
```

Comment:
I think here we should always have a `token` entity created.

Reply:
I'll close this since the `Pair` entity has been removed.